docs/sampling/README.md

```diff
@@ -0,0 +1,262 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Sampling
+--->
+
+# Semantic Conventions for Sampling
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+
+- [Probability sampling](#probability-sampling)
+  - [Overriding sampling decisions](#overriding-sampling-decisions)
+  - [Overriding sampling randomness](#overriding-sampling-randomness)
+  - [Sampling threshold](#sampling-threshold)
+  - [Sampling randomness](#sampling-randomness)
+  - [No definition for Scope and Resource attributes](#no-definition-for-scope-and-resource-attributes)
+- [Span sampling attributes](#span-sampling-attributes)
+- [Logs sampling attributes](#logs-sampling-attributes)
+- [Examples](#examples)
+  - [Head sampling](#head-sampling)
+  - [Tail sampling](#tail-sampling)
+
+<!-- tocstop -->
+
+These attributes reflect the effect of sampling in a telemetry
+collection pipeline.  These attributes describe how items of telemetry
+were collected, making it possible for Span-to-Metrics pipelines
+and Logs-to-Metrics pipelines to accurately count Spans and Log
+Records of telemetry, before sampling, in a probabilistic sense.
+
+These attributes MAY be modified by components in a collection
+pipeline to convey successive sampling that has been carried out for a
+particular item of telemetry, using the conventions for consistent
+sampling described here.  In that sense, telemetry consumers
+should see these attributes as telemetry metadata.
+
+## Probability sampling
+
+The OpenTelemetry sampling decision is defined in terms of a Threshold
+value and a Randomness value, each containing 56 bits of information.
+
+A constant known as _maximum adjusted count_ (`MaxAdjustedCount`),
+with value `0x100000000000000`, (which can also be expressed
+`0x1p+56`, `math.Pow(2, 56)`, or `math.Ldexp(1, 56)`), defines the
+exclusive upper limit of these values.
+
+Logically, both Threshold (`T`) and Randomness (`R`) are represented
+as unsigned integers in the range `0` through `0xffffffffffffff` or
+`MaxAdjustedCount - 1`.  Items of telemetry are selected (i.e.,
+"sampled") when their Threshold value is less than or equal to their
+Randomness value, or `T <= R`.
+
+Sampling probability is defined by the following expression:
+
+```
+Probability = (MaxAdjustedCount - Threshold) / MaxAdjustedCount
+```
+
+In a Span-to-Metrics or Logs-to-Metrics pipeline, each item of
+telemetry is representative of an _adjusted count_ number of items in
+the original population.  Adjusted count is the inverse of sampling
+probability, and `MaxAdjustedCount` (defined above) is the inverse of
+the smallest supported sampling probability (which can also be
+represented as `0x1p-56`, `math.Pow(2, -56)`, or `math.Ldexp(1,
+-56)`).
```

Comment:
The above is a bit inaccurate (the smallest supported sampling probability is actually 1/(MaxAdjustedCount-1), I suggest removing the second part of the sentence.

Reply:
The smallest sampling probability is `2^-56`, which is realized with a threshold of `2^56-1 = MaxAdjustedCount - 1`. So I think the proposal is correct.

Reply:
You are right. I cannot count correctly ...